### ISSUE TITLE
Multiline image caption

### ISSUE THREAD

**Issue by zetrixweb** [ishaq/SimplePDF#20]

**Comment by ishaq** [comment]
yeah, currently image captions are single line because they are drawn using `CTLine` primitive. I'll put it on TODOs. For now, you can output multi-column text before outputting images to make it look as if images are preceded by multi-line captions.

**Comment by Razie7** [comment]
Hi Zetrixweb, 
Hi 
Could you please share how to get this working?  No such Module "SimplePDFSwift" although I successfully installed SimplePDFSwift 
That happened to me when I used the example file and when I started new project

I use:
Xcode 7.3, CocoaPod 1.0.0

Thank you!
